docs/en/remote-flow.rst

```diff
@@ -303,14 +343,12 @@ The JWT payload parameters are described herein:
 
   * - **Name**
```

Comment:
wallet_nonce is missing

Reply:
wallet_nonce is optional, I have added it anyway.